macros/models/_sector/tokens/transfers_enrich.sql

```diff
@@ -21,7 +21,7 @@ SELECT t.blockchain
 , t."from"
 , t.to
 , t.contract_address
-, {{case_when_token_standard("'{{native_symbol}}'", 'tokens_erc20.symbol', 'NULL')}} AS symbol
```

Comment:
ah it's not possible to do `{{}}` inside of `{{}}` already, @jeff-dude do you know what to do here?

Reply:
s/o to @tomfutago for this idea in some of the latest sector spell designs

Reply:
nice thank you!